x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/81_text_exact_subfields.yml

```diff
@@ -239,11 +252,15 @@ setup:
   - match: { columns.5.name: "non_indexed.raw" }
   - match: { columns.5.type: "keyword" }
 
-  - length: { values: 1 }
```

Comment:
@luigidellaquila I'd appreciate it if you could take a look at the changes in this test file.  This PR changes the "exact subfield" behavior, and I want to make sure the ESQL team is aware of that change.  Happy to discuss if you have any questions or concerns.